AK/Demangle.h

```diff
@@ -21,6 +25,12 @@ inline ByteString demangle(StringView name)
         free(demangled_name);
     return string;
 }
+#else
+inline ByteString demangle(StringView name)
+{
+    return name;
```

Comment:
Stubs are fine, but can we get a `// FIXME` comment here to say what needs doing? That'll make it easier to find later. (Maybe even print a log message.)